README.md

```diff
@@ -1,2 +1,171 @@
-# football-leagues-standings
-A mobile web page that shows information about standings of Football Leagues. Built with React and Redux.
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1><b>
+World Countries Information</b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [🌍 World Countries Information ](#-world-countries-information-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 🌍 World Countries Information <a name="about-project"></a>
+
+World Countries Information is a mobile web application based in the [RestCountry API](https://restcountries.com/). Users can search for countries and see population stats and more detailed information like capitals, time zones, currencies, languages and more.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- React
+- Redux
+- Tailwind Css
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <li><a href="https://tailwindcss.com/">Tailwind Css</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Fetches a list of countries from [RestCountry API](https://restcountries.com/) and shows population stats.
+- Fetches a detailed list of information about a selected country.
+- User can search by Country name and region.
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> You can see the Live Demo [here.](https://world-countries-information.onrender.com)
+
+> You can watch the video description of the project [here.](https://www.loom.com/share/9359264f44994a5fb7919a982cd1817a?sid=9df1a396-0aa2-488b-8c4d-26461fcdf933)
```

Comment:
- [ ] As part of the requirements, you should also highlight some interesting piece of code or something you built that you are very proud of while presenting the video of which  is missing in yours. Kindly make a new one add talk about that 👍 

### Instructions:
![image](https://github.com/Luiscarlosvd/world-countries-info/assets/50990925/5047a5a2-352e-49bb-8cc4-d2eb43f90489)